src/promptflow-evals/promptflow/evals/evaluate/_code_client/proxy_client.py

```diff
@@ -0,0 +1,34 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import logging
+
+import numpy as np
+
+from promptflow.client import PFClient
+from promptflow.tracing import ThreadPoolExecutorWithContext as ThreadPoolExecutor
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ProxyRun:
```

Comment:
Do we want to add some utility functions to this class in future?

Reply:
Synced with Nikolay offline. We will need this proxy for now so we can easily switch between the code client and the proxy client. Another reason for needing it is due to the BatchRunContext. We can remove this layer once we have proven that pf.run meets our expectations and we no longer need the code client.